src/result_processor.c

```diff
@@ -452,7 +452,9 @@ static int rpsortNext_innerLoop(ResultProcessor *rp, SearchResult *r) {
                                     .status = &status};
       RLookup_LoadDocument(NULL, &h->rowdata, &loadopts);
       if (QueryError_HasError(&status)) {
-        return RS_RESULT_ERROR;
+        // failure to fetch the doc. ignore it and continue 
+        rp->parent->totalResults--;
+        return RESULT_QUEUED;
```

Comment:
Thinking about it now, what if someone will do sortby on a none existing field? Before he will get an error and now we will skip the document?

Reply:
That is true.
Lets let it get into the queue. and we should add a test for it

Reply:
Summery of the discussion:
If the field does not exists the `loadIndividualKeys` would have return REDISMODULE_OK and will not return with error. So it is not a breaking change.